### DISCUSSION TITLE
【Help】 How to decode smartfoxserver message.

### DISCUSSION THREAD

**Discussion by Stephen0910** [mitmproxy/mitmproxy#7313] [Intercept Everything!]
https://docs2x.smartfoxserver.com/Overview/sfs2x-protocol
Python is not supported in smartfox official api. I have tried many ways to decode the message, but failed. Is anyone able to help with any solution? It's so important for me. Great thanks.

**Comment by a0w89fwa0f** [comment]
Were you able to figure this out @Stephen0910 ?

**Reply by Stephen0910** [reply]
> Were you able to figure this out @Stephen0910 ?

[pyfox2x](https://github.com/MSM-Hacks/pyFox2X)
